release/_bom.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+function download_bnd_files {
+	lc_cd "${_BUNDLES_DIR}/deploy"
+
+	if [ ! -e "${_BUNDLES_DIR}/osgi/modules/biz.aQute.remote.agent-6.4.0.jar" ]
+	then
+		lc_download "https://repo1.maven.org/maven2/biz/aQute/bnd/biz.aQute.remote.agent/6.4.0/biz.aQute.remote.agent-6.4.0.jar" "biz.aQute.remote.agent-6.4.0.jar"
+	fi
+
+	lc_cd "${_BUILD_DIR}/boms"
+
+	rm -f ./*.jar
```

Comment:
@pgydev, what JAR files do you need to remove? Won't this command delete unintentional files? Can you please double check and remove just what is needed to be removed?